__tests__/app.test.js

```diff
@@ -402,3 +402,55 @@ describe("Task 10 - GET /api/users", () => {
       });
   });
 });
+
+describe("Task 11 - GET /api/articles (topic query)", () => {
+  test("should return all the articles with the queried topic", () => {
```

Comment:
we should also have a test for when the topic exists but has no articles associated with it